salt/modules/grafana4.py

```diff
@@ -183,7 +183,7 @@ def create_user(profile='grafana', **kwargs):
     return response.json()
 
 
-def update_user(userid, profile='grafana', **kwargs):
+def update_user(userid, profile='grafana', orgid=None, **kwargs):
```

Comment:
IMHO, we should consider making this `org_id` instead to make it easier to identify.

Reply:
If I change it here, for consistency, I'd probably better change this anywhere. (orgid->org_id, userid->user_id, orgname->org_name)
This would lead to the 117 changes and is potential breaking change:
```
# git diff --numstat
105     105     salt/modules/grafana4.py
7       7       salt/states/grafana4_dashboard.py
6       6       salt/states/grafana4_datasource.py
5       5       salt/states/grafana4_user.py
```

go/no-go?

Reply:
Good point. I agree with you then. Let's leave it. :)